dev-requirements.txt

```
@@ -1,8 +1,10 @@
 black[jupyter]~=22.3.0
-flake8==3.8.4
 flake8-import-order==0.18.1
-mypy==0.910
-pylint==2.6.0
-pytest==6.1.2
+flake8~=3.8.4
+mypy~=0.961
```

nit
```suggestion
mypy>=0.961
```
(they're synonymous in this case but imo `>=` is more clear)